### PR TITLE
parent_select plugin: look in ats config dir for yaml config if path is relative

### PR DESCRIPTION
**Pull request by traeak**
Also fixes documentation error on how to use the plugin.

### CODE REVIEW

File: plugins/experimental/parent_select/parent_select.cc [hunk 1]
```diff
@@ -22,12 +22,13 @@
...
+#include <cstdlib>
+#include <cstring>
+#include <filesystem>
+#include <fstream>
 #include <iostream>
 #include <map>
 #include <memory>
-#include <fstream>
-#include <cstdlib>
-#include <cstring>
```
[the remark this answers is not in the document]
Comment (traeak) [reply plugins/experimental/parent_select/parent_select.cc:30]: is this a bad thing?

**Review by ywkaras — APPROVED** [review]

**Comment by traeak** [comment]
[approve ci autest]

**Comment by traeak** [comment]
[approve ci autests]

**Comment by traeak** [comment]
[approve ci autest 2of4]

**Comment by traeak** [comment]
[approve ci autest]

**Pull request merged**